.travis.yml

```diff
@@ -14,9 +15,20 @@ env:
   - ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
   - ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
 
+matrix:    # Laravel 5.6 & 5.7 do not support PHP7.0
+  exclude:
+    - php: 7.0
+      env: ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
+    - php: 7.0
+      env: ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
+
 before_install:
-  - travis_retry composer self-update
-  - travis_retry composer update --no-interaction --prefer-source
+  - composer require illuminate/support:${ILLUMINATE_VERSION} illuminate/validation:${ILLUMINATE_VERSION} --no-update
```

Comment:
There is no need to require the `illuminate/validation` dependency instead the `illuminate/http` should be included